app/code/local/LoyaltyLion/Core/Model/Observer.php

```diff
@@ -6,21 +6,31 @@ class LoyaltyLion_Core_Model_Observer {
   private $session;
 
   public function __construct() {
-    $this->token = Mage::getStoreConfig('loyaltylion/configuration/loyaltylion_token');
-    $this->secret = Mage::getStoreConfig('loyaltylion/configuration/loyaltylion_secret');
-
-    if (!$this->isEnabled()) return;
```

Comment:
There is no point in doing this, checking if we're enabled here doesn't help us as we will potentially have the wrong store in scope.